libs/emoji.js

```diff
@@ -233,9 +233,940 @@ var emoji = (function() {
+    "1f6a1": { "sheet": 10, "x": 37 },
+    "1f6a3": { "sheet": 10, "x": 38 },
+    "1f6a6": { "sheet": 10, "x": 39 },
+    "1f6a8": { "sheet": 10, "x": 40 },
+    "1f6a9": { "sheet": 10, "x": 41 },
+    "1f6aa": { "sheet": 10, "x": 42 },
+    "1f6ab": { "sheet": 10, "x": 43 },
+    "1f6ae": { "sheet": 10, "x": 44 },
+    "1f6af": { "sheet": 10, "x": 45 },
+    "1f6b0": { "sheet": 10, "x": 46 },
+    "1f6b1": { "sheet": 10, "x": 47 },
+    "1f6b3": { "sheet": 10, "x": 48 },
+    "1f6b4": { "sheet": 10, "x": 49 },
+    "1f6b5": { "sheet": 10, "x": 50 },
+    "1f6b7": { "sheet": 10, "x": 51 },
+    "1f6b8": { "sheet": 10, "x": 52 },
+    "1f6bf": { "sheet": 10, "x": 53 },
+    "1f6c1": { "sheet": 10, "x": 54 },
+    "1f6c2": { "sheet": 10, "x": 55 },
+    "1f6c3": { "sheet": 10, "x": 56 },
+    "1f6c4": { "sheet": 10, "x": 57 },
+    "1f6c5": { "sheet": 10, "x": 58 },
+
+    "1f1ee-1f1f3": { "sheet": 12, "x": 0 },
+    "1f1f2-1f1fd": { "sheet": 12, "x": 1 },
+    "1f1e7-1f1f7": { "sheet": 12, "x": 2 },
+    "1f1f8-1f1e6": { "sheet": 12, "x": 3 },
+    "1f1ff-1f1e6": { "sheet": 12, "x": 4 },
+    "1f1e6-1f1f7": { "sheet": 12, "x": 5 },
+    "1f1f3-1f1f1": { "sheet": 12, "x": 6 },
+    "1f1f9-1f1f7": { "sheet": 12, "x": 7 },
+    "1f1f2-1f1fe": { "sheet": 12, "x": 8 },
+    "1f1fb-1f1ea": { "sheet": 12, "x": 9 },
+    "1f1e8-1f1f4": { "sheet": 12, "x": 10 },
+    "1f1e8-1f1f1": { "sheet": 12, "x": 11 },
+    "1f1ed-1f1f0": { "sheet": 12, "x": 12 },
+    "1f1f3-1f1ec": { "sheet": 12, "x": 13 },
+    "1f1e8-1f1ed": { "sheet": 12, "x": 14 },
+    "1f1ee-1f1f1": { "sheet": 12, "x": 15 },
+    "1f1f9-1f1ed": { "sheet": 12, "x": 16 },
+    "1f1f8-1f1ec": { "sheet": 12, "x": 17 },
+    "1f1e6-1f1ea": { "sheet": 12, "x": 18 },
+    "1f1f9-1f1fc": { "sheet": 12, "x": 19 },
+    "1f1ea-1f1ec": { "sheet": 12, "x": 20 },
+    "1f1e8-1f1e6": { "sheet": 12, "x": 21 },
+    "1f1f2-1f1e8": { "sheet": 12, "x": 22 },
+    "1f1e6-1f1f9": { "sheet": 12, "x": 23 },
+    "1f1e6-1f1fa": { "sheet": 12, "x": 24 },
+    "1f1e7-1f1e6": { "sheet": 12, "x": 25 },
+    "1f1e7-1f1ea": { "sheet": 12, "x": 26 },
+    "1f1e8-1f1ee": { "sheet": 12, "x": 27 },
+    "1f1e8-1f1f2": { "sheet": 12, "x": 28 },
+    "1f1e8-1f1f7": { "sheet": 12, "x": 29 },
+    "1f1e9-1f1ff": { "sheet": 12, "x": 30 },
+    "1f1ea-1f1e8": { "sheet": 12, "x": 31 },
+    "1f1ec-1f1ed": { "sheet": 12, "x": 32 },
+    "1f1ec-1f1f7": { "sheet": 12, "x": 33 },
+    "1f1ed-1f1f3": { "sheet": 12, "x": 34 },
+    "1f1ed-1f1f7": { "sheet": 12, "x": 35 },
+    "1f1ee-1f1f7": { "sheet": 12, "x": 36 },
+    "1f1ef-1f1f4": { "sheet": 12, "x": 37 },
+    "1f1f0-1f1ff": { "sheet": 12, "x": 38 },
+    "1f1f1-1f1e7": { "sheet": 12, "x": 39 },
+    "1f1f5-1f1ea": { "sheet": 12, "x": 40 },
+    "1f1f5-1f1f9": { "sheet": 12, "x": 41 },
+    "1f1f8-1f1fe": { "sheet": 12, "x": 42 },
+    "1f1fa-1f1e6": { "sheet": 12, "x": 43 },
+    "1f1fa-1f1fe": { "sheet": 12, "x": 44 },
+    "1f1fd-1f1ea": { "sheet": 12, "x": 45 },
```

Comment:
I know this is pre-existing, but why are all the "flag" emoji blank in this sheet?